### DISCUSSION TITLE
[Lesson 2] [Concept 3] [Privacy Dilemma] Definition of privacy based on what society thinks

### DISCUSSION THREAD

**Discussion by late-warrior** [OpenMined/courses#129] [Course: Our Privacy Opportunity] [answered]
[Video link](https://courses.openmined.org/courses/our-privacy-opportunity/1f34d3a8-2fa1-4a1e-9ef1-eaaf9dddd617/5f2cf924-0bc5-4f5b-a347-452dd1e0ab84) 

Lesson 2 is an interesting take on privacy - no absolute definition but that everything in context of what 'society' thinks..   
Does that mean the definition of privacy will differ between countries and cultures (and probably education levels ) ?  For eg. an app (in India) promised a cashback offer if you uploaded all your contacts and one of them booked and all my close family who live in smaller towns happily gave them away..  Very few of the more 'aware' folks (mostly folks who worked in IT in cities) did this..

Awesome course and great work the entire OpenMined team.  The passion literally oozes from from the videos.

[Reposting from Slack]

**Comment by em-blue** [comment, marked as answer]
Hey @late-warrior ! Thanks for watching the course!
The work behind contextual integrity explains it in a way that's a bit more nuanced than that, but basically, the goal was to pinpoint what people considered to be "a privacy invasion", because it's actually quite complicated. In the course, we've called them "social norms" but the real term is "context-relative social norms" in Helen's book. 

One interesting thing is that what is considered "a social norm" right now is not necessarily "best". For example, if the social norm is that people generally aren't very bothered about their location being shared/collected, but in an ideal world they really probably _should be_ bothered about that, then perhaps a more ideal information flow actually isn't in line with 'social norms', but instead creating what is actually ideal.

There are also some other definitions of privacy, of course - if you're interested in reading more, I'd recommend "What do we know and what can we do about Internet Privacy" by Paul Bernal - it's a really short and friendly book!
If you're interested in an academic deep-dive, read Helen's work on Privacy in Context (and more recent papers on contextual integrity!)

**Reply by Hitesh111** [reply]
But how would we define what is ideal and what isn't? and how do we differentiate ideals and social norms?

**Reply by em-blue** [reply]
To the first question: That's a question that lawmakers, societies, communities, and philosophers, etc have been discussing for a very long time!

To the second question: I think you'll really enjoy Helen's book Privacy in Context. She's a philosopher and goes into that aspect of things!

**Comment by kr7p0** [comment]
Yes, the definition of "privacy" can differ widely across countries and education levels, especially when it comes to technology and apps asking for sensitive data like contact uploads. In India, privacy awareness is growing, but many students and users may not fully understand the risks of sharing personal contacts with apps. Cultural attitudes, legal frameworks like India’s Personal Data Protection Bill, and educational exposure to digital rights play major roles. In contrast, countries with stricter privacy laws (like GDPR in Europe) tend to enforce more transparent data practices. It’s important for students to stay informed about digital privacy and make cautious choices. For reliable career guidance, course options, and detailed insights about digital literacy, visit [StudentSaathi](https://studentsaathi.com/)

**Reply by cloudmicrotech-eng** [reply]
Foundations of Priva
